src/main/java/hudson/plugins/sidebar_link/ProjectLinks.java

```diff
@@ -44,7 +44,9 @@
 
     @DataBoundConstructor
     public ProjectLinks(List<LinkAction> links) {
-        this.links = links;
+        if (links != null) {
+            this.links = links;
+        }
```

Comment:
I will assign an empty list as Oleg suggested. The default empty list does not guarantee that it is impossible to assign null. It happens to me after loading config.xml - I know that readResolve prevents this case, but adding of else costs nothing and make it less breakable.